packages/dds/merge-tree/src/test/beastTest.ts

```diff
@@ -1249,7 +1239,7 @@ export function TestPack(verbose = true) {
 				for (let j = 0; j < removeCount; j++) {
 					const dlen = randTextLength();
 					const preLen = cliA.getLength();
-					const pos = random.integer(0, preLen)(mt);
+					const pos = random.integer(0, preLen - dlen);
```

Comment:
previous random removal bounds generation here was bogus. switching the RNG source from mersenne twister to XSAdd exposed this issue.